internal/pkg/utils/testhelper/testtemplateinputs/testtemplateinputs.go

```diff
@@ -0,0 +1,33 @@
+package testtemplateinputs
+
+import (
+	"context"
+	"strings"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/env"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/testhelper"
+)
+
+type testInputsEnvProvider struct {
+	ctx  context.Context
+	envs *env.Map
+}
+
+// CreateTestInputsEnvProvider allows you to inject only ENV variables with correct prefix.
+func CreateTestInputsEnvProvider(ctx context.Context) (testhelper.EnvProvider, error) {
+	allEnvs, err := env.FromOs()
+	if err != nil {
+		return nil, err
+	}
+	res := make(map[string]string)
+	for k, v := range allEnvs.ToMap() {
+		if strings.HasPrefix(k, "KAC_SECRET_") {
+			res[k] = v
+		}
+	}
+	return &testInputsEnvProvider{ctx: ctx, envs: env.FromMap(res)}, nil
+}
```

Comment:
V tomto nevidim velmi zmysel, ... filtrovanie secrets mas uz v github actions.
Nie kazda ENV co mozem chciet nahradit musi byt secret, moze to byt aj nejaka casto sa opakujuca hodnota.

Reply:
dobře.